script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -164,38 +175,39 @@ I walk up the stairs in the lobby.
 [sha:99b82c823d1d2b7594c0fa9ba008b9c02d00197d]{
 -- Page 363, Offset 15690.
 -- 豪華なシャンデリアを横目に楽しみながら、西館二階の端にある自室に足を向けた。
-I enjoy the sight of the gorgeous chandeliers out of the corner of my eyes as I head toward my room on the second floor of the west wing.
+I enjoy the sight of the gorgeous chandeliers out of the corner of my eye as I head toward my room on the second floor of the west wing.
 }
 [sha:cceb047af00da5deddd74d1f7eb1853592edbe38]{
 -- Page 364, Offset 15691.
 -- 「あ、お帰り～♪」
-"Oh, welcome home. ♪"
+"Oh, welcome hooome ♪"
 }
 [sha:eca3c59f10971d3ac97b50b35b67e6c95087d7da]{
 -- Page 364, Offset 15692.
 -- 「な／に／ぃ／！？」
 // weird formatting used in the Japanese here.
-"Wh-a-t?!"
+"Wh―a―t?!"
 }
 [sha:e639d1deb27a198cccafa29fb20e5fec2bb5fad3]{
 -- Page 364, Offset 15693.
 -- 朝のシエル先輩を凌駕するサプライズが、ベッドに腰掛けて足をブラブラさせていた。
-On my bed, swinging its legs back and forth, sits a surprise exceeding even that of this morning's Ciel.
+An even bigger surprise than seeing Ciel-senpai this morning is sitting on my bed,
```

Comment:
prefer prev version

Reply:
I guess the previous version needs the -senpai part at least though, but yes.